packages/core/content-manager/server/src/services/document-manager.ts

```diff
@@ -8,7 +8,7 @@ import { sumDraftCounts } from './utils/draft';
 import { ALLOWED_WEBHOOK_EVENTS } from '../constants';
 
 type DocService = Modules.Documents.ServiceInstance;
-type DocServiceParams<TAction extends keyof DocService> = Parameters<DocService[TAction]>;
+type DocServiceParams<TAction extends keyof DocService> = Parameters<DocService[TAction]>[0];
```

Comment:
utils like this would be very useful internally for us but also for plugin developers, do you think it could be a good idea to export it somehow?

e.g Modules.Documents.Params<Taction>

Reply:
Yeah I would like to rework the typings a bit later